pkg/machines/components/storagePools/storagePool.jsx

```diff
@@ -143,15 +150,26 @@ class StoragePoolActions extends React.Component {
                 {_("Delete")}
             </Button>
         );
-
+        let deleteBtnWrapper;
         if (storagePool.active) {
-            return (
+            deleteBtnWrapper = (
                 <OverlayTrigger overlay={ <Tooltip id='tip-pool-delete'>{_("Try deactivating the storage pool before attempting deletion")}</Tooltip> } placement='top'>
                     {deleteBtn}
                 </OverlayTrigger>
             );
         } else {
-            return deleteBtn;
+            deleteBtnWrapper = deleteBtn;
         }
+
+        return (
+            <React.Fragment>
+                { !storagePool.active &&
+                <Button id={`activate-${id}`} onClick={this.onActivate.bind(this)}>
+                    {_("Activate")}
```

Comment:
What was the reason to name this operation `Activate` instead of `Start`?

When we create storage pool in cockpit there is an option `Startup` with description `Start pool when host boots` so for consistency we should use `Start` here as well.

Reply:
I made this decision according to the state wording. State is active/inactive not started/stopped. However I don't have strong opinion on it, I am ok to change this to match the other sentence you mentioned.

Reply:
Please no `.bind()` call in `render()`, do this in the constructor instead to only do it once instead of once per rendering.